backend/store_db.py

```diff
@@ -0,0 +1,38 @@
+import json
+from pymongo import MongoClient
+
+# MongoDB connection string
+atlas_connection_uri = "mongodb+srv://gen:gen123@cluster0.bqucodi.mongodb.net/?retryWrites=true&w=majority"
+
+# Connect to MongoDB Atlas
+client = MongoClient(atlas_connection_uri)
+
+# Database name
+db = client.open_montreal
+
+# GeoJSON file paths and their corresponding collection names
+geojson_files = {
+    "C:/Users/korjo/Downloads/pdq.geojson": "pdq"
```

Comment:
its the absolute here but we want the relative

Reply:
changed the code